src/DB_Command.php

```diff
@@ -554,6 +554,16 @@ public function export( $args, $assoc_args ) {
 
 		$support_column_statistics = exec( $mysqldump_binary . ' --help | grep "column-statistics"' );
 
+		/**
+		 * Issue #144
+		 * In case that `--default-character-set` is not given and `DB_CHARSET` is `utf8`,
+		 * use `utf8mb4` as a default-character-set to show emoji.
+		 */
```

Comment:
```suggestion
		/*
		 * In case that `--default-character-set` is not given and `DB_CHARSET` is `utf8`,
		 * use `utf8mb4` as a `default-character-set` to ensure emojis are encoded correctly.
		 */
```